task_DEV-9/ConsoleHandler.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace task_DEV_9
+{
+  public class ConsoleHandler

```

Comment:
Very strange name for class, no comments - what is the responsibility of the class